--- LOCAL:am_util_pmu.c
+++ MERGED:am_util_pmu.c
@@ -3,19 +3,45 @@
 //! @file am_util_pmu.c
 //!
 //! @brief The Performance Monitors Extension describes an optional non-invasive
-//! component that allows performance events to be identified and countered.
-//! This enables software to get information about events that are taking place
-//! in the processor and can be used for performance analysis and system debug.
 //!
 //! The PMU supports eight 16-bit event counters and one 32-bit cycle counter.
-//! The PMU also supports a chain function which allows the PMU to cascade two
-//! of the 16-bit counters into one 32-bit counter.only odd event counter support
-//! the chain feature. That means by chaining counters in pairs, the counter
-//! range can be increased by halving the number of counters.
-//!
-//! @addtogroup pmu PMU Functionality
 //! @ingroup utils
 //! @{
+//
+//*****************************************************************************
+
+//*****************************************************************************
+//
+// Copyright (c) 2025, Ambiq Micro, Inc.
+// All rights reserved.
+//
+// Redistribution and use in source and binary forms, with or without
+// modification, are permitted provided that the following conditions are met:
+//
+// 1. Redistributions of source code must retain the above copyright notice,
+// this list of conditions and the following disclaimer.
+//
+// 2. Redistributions in binary form must reproduce the above copyright
+// notice, this list of conditions and the following disclaimer in the
+// documentation and/or other materials provided with the distribution.
+//
+// 3. Neither the name of the copyright holder nor the names of its
+// contributors may be used to endorse or promote products derived from this
+// software without specific prior written permission.
+//
+// Third party software included in this distribution is subject to the
+// additional license terms as defined in the /docs/licenses directory.
+//
+// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+// AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+// IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+// ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+// LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+// CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+// SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+// INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+// CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+// ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 //
 //*****************************************************************************
 
@@ -53,7 +79,11 @@
 // ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 // POSSIBILITY OF SUCH DAMAGE.
 //
+<<<<<<< LOCAL
 // This is part of revision release_sdk5p0p0-5f68a8286b of the AmbiqSuite Development Package.
+=======
+// This is part of revision release_sdk5_2_a_1_1-c2486c8ef of the AmbiqSuite Development Package.
+>>>>>>> REMOTE
 //
 //*****************************************************************************
 #include <stdint.h>
